nexus/db-model/src/console_session.rs

```diff
@@ -4,13 +4,18 @@
 
 use chrono::{DateTime, Utc};
 use nexus_db_schema::schema::console_session;
+use omicron_uuid_kinds::ConsoleSessionKind;
+use omicron_uuid_kinds::TypedUuid;
 use uuid::Uuid;
 
+use crate::typed_uuid::DbTypedUuid;
+
 // TODO: `struct SessionToken(String)` for session token
 
 #[derive(Queryable, Insertable, Clone, Debug, Selectable)]
 #[diesel(table_name = console_session)]
 pub struct ConsoleSession {
+    pub id: DbTypedUuid<ConsoleSessionKind>,
```

Comment:
part of me wonders whether we should just have the type [derive `Asset`](https://github.com/oxidecomputer/omicron/blob/7547bfa56e309926a416ac90c18ff310f2b1f045/nexus/db-macros/src/lib.rs#L167-L178) instead, but we don't have a `time_modified`/`time_deleted` field and it's probably not worth adding those just to be able to use the derive...